Beatmap/src/BeatmapFromKSH.cpp

```diff
@@ -217,7 +217,7 @@ AudioEffect ParseCustomEffect(const KShootEffectDefinition& def)
 				MultiParamRange pr = { ParseParam(a), ParseParam(b) };
 				if (pr.params[0].type != pr.params[1].type)
 				{
-					Logf("Non matching parameters types \"%s\" for key: %s", Logger::Warning, s, s.first);
+					Logf("Non matching parameters types \"[%s, %s]\" for key: %s", Logger::Warning, s.first, s.second, s.first);

```

Comment:
How was this printed out originally? It was causing errors and I had to change it, but wasn't sure how it looked before.

Reply:
I think it should be
`Logf("Non matching parameters types \"%s\" for key: %s", Logger::Warning, s.second, s.first);`